kube-scheduler-healthcheck

```diff
@@ -2,12 +2,14 @@
 set -eu
 TEST_POD_NAMESPACE="${TEST_POD_NAMESPACE:-kube-system}"
 TEST_POD_IMAGE="${TEST_POD_IMAGE:-gcr.io/google_containers/pause:3.0}"
+TEST_POD_NAME_POSTFIX="${TEST_POD_NAME_POSTFIX-''}"
+TEST_POD_NAME="scheduler-pod-healthcheck-${TEST_POD_NAME_POSTFIX}"
```

Comment:
this will leave a dangling `-` on the end of the pod if the variable isn't set

Could set a default postfix other than empty string

Reply:
good postfix recommendation?

Reply:
/shrug
```
TEST_POD_NAME_POSTFIX="${TEST_POD_NAME_POSTFIX-'pod'}"
TEST_POD_NAME="scheduler-healthcheck-${TEST_POD_NAME_POSTFIX}"
```